dynamodb/src/it/scala/zio/dynamodb/TypeSafeApiCrudSpec.scala

```diff
@@ -95,6 +108,20 @@ object TypeSafeApiCrudSpec extends DynamoDBLocalSpec {
           } yield assertTrue(p == person)
         }
       },
+      test("big case class simple round trip") {
+        withSingleIdKeyTable { tableName =>
+          // format: off
+          val big = Big(
+            "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20",
+            "21", "22", "23", "24", "25", "26", "27", "28", "29", "30", Some("31"), Some("32")
```

Comment:
I know that you have tests elsewhere for the null behavior with None, but do you think that it makes sense to have one of the options be a None here so the encoding behavior for None is tested through the "whole" lifecycle?

What about adding a field that is a map and asserting that passing in a `Map.empty` will also work?

Reply:
That's tested here, the way `puts` work.